doc/source/categorical.rst

```diff
@@ -766,6 +766,14 @@ Dtype comparisons work:
     dtype == np.str_
     np.str_ == dtype
 
+To check if a Series contains Categorical data, with pandas 0.16 or later, use
+``hasattr(s, 'cat')``:
+
+.. ipython:: python
```

Comment:
can you add a `:okexcept:` option to the directive here (so the doc building does not issue a warning for the failure)?